app/controllers/configuration_controller.rb

```diff
@@ -662,20 +662,20 @@ def build_tabs
     @breadcrumbs = Array.new
     case @config_tab
     when "ui"
-      @tabs = Array.new
       drop_breadcrumb( {:name=>"User Interface Configuration", :url=>"/configuration/edit" }, true ) if @tabform != "ui_4"
       case @tabform
       when "ui_1"
-        @tabs[0] = ["1", ""]  # Start with first tab array entry set to tab 1 as active
+        @active_tab = "1"
       when "ui_2"
-        @tabs[0] = ["2", ""]  # Start with first tab array entry set to tab 1 as active
+        @active_tab = "2"
       when "ui_3"
-        @tabs[0] = ["3", ""]  # Start with first tab array entry set to tab 1 as active
+        @active_tab = "3"
       when "ui_4"
-        @tabs[0] = ["4", ""]  # Start with first tab array entry set to tab 1 as active
+        @active_tab = "4"
       when "ui_5"
-        @tabs[0] = ["5", ""]  # Start with first tab array entry set to tab 1 as active
+        @active_tab = "5"
       end
```

Comment:
@matthewd Do we want to replace the case with something like `@active_tab = @tabform.split("_").last` at this time?  One more place below as well.